.gitignore

```diff
@@ -9,4 +9,5 @@ MANIFEST
 /static/
 .DS_Store
 keel/.DS_Store
-
+env/
+*db.sqlite3
```

Comment:
Do wee need this?

Reply:
I created a virtual env on my local, added to it git ignore so it won't be staged